beast/tests/beast_settings_for_tests.txt

```diff
@@ -1,229 +0,0 @@
-# """ Data Model interface v3.0
```

Comment:
I was a bit confused by this at first.  Was it supposed to be deleted as part of #612?

Reply:
Yes.  But this likely a bit of leftover from me branching from a branch.  And then rebasing a couple of times to get things cleanly compared to the master.  As long at it is just a removal, maybe it will be ok.